### PR TITLE
Fix failing tests for braveryPanel

### PR DESCRIPTION
**Pull request by cezaraugusto**
Auditors: @bbondy
Fix: #9059 
Close: #8760

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:

```
npm run test -- --grep="Bravery Panel Adblock stats without iframe tests block device enumeration"
npm run test -- --grep="navigator component tests lion badge is grayed out if shield is disabled"
npm run test -- --grep="Adblock stats iframe tests"
```

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: test/bravery-components/braveryPanelTest.js [hunk 1]
```diff
@@ -882,29 +882,6 @@ describe('Bravery Panel', function () {
               return body === ''
             })
         })
-
-        .keys(Brave.keys.ESCAPE)
```
Comment (cezaraugusto) [inline test/bravery-components/braveryPanelTest.js]: introduced here: https://github.com/brave/browser-laptop/pull/8990/files#diff-f94f0ea4252431046502bbe75728ba1fR836. I don't think it is needed for this test specifically

**Comment by luixxiul** [comment]
the first one seems to be still failing on travis: https://travis-ci.org/brave/browser-laptop/jobs/236166580#L3120

````
  1) Bravery Panel Adblock stats without iframe tests block device enumeration:
     WaitUntilTimeoutError: Promise was rejected with the following reason: timeout
      at getText("body") - braveryPanelTest.js:867:23
````

**Comment by cezaraugusto** [comment]
it is passing locally, seems intermittent. I'll change some methods to make Travis happy

File: test/bravery-components/braveryPanelTest.js (hunk 1 above)
Comment (luixxiul) [inline test/bravery-components/braveryPanelTest.js]: thanks!

**Comment by luixxiul** [comment]
@cezaraugusto if you are in the code, would you mind checking https://github.com/brave/browser-laptop/issues/8760#issuecomment-301157088 ?

**Comment by cezaraugusto** [comment]
@luixxiul added

**Comment by luixxiul** [comment]
the last commit fixes #8760 but enumeration stuff is still there :-(

**Comment by cezaraugusto** [comment]
I'm closing this in favor of #9148 and #9147

**Pull request closed without merging**